components/ee/ws-scheduler/pkg/scheduler/scheduler.go

```diff
@@ -342,8 +342,10 @@ func (s *Scheduler) buildState(ctx context.Context, pod *corev1.Pod) (state *Sta
 		return nil, err
 	}
 
-	// We need to take _all_ pods into account to accurately calculate available RAM per node
-	allPods, podsErr := s.pods.Lister().List(labels.Everything())
+	// We need to take into account _all_ pods in _all_ namespaces to accurately calculate available RAM per node
+	// NOTE: .Pods("") - in contrast to omitting it and calling .Lister().List(...) directly - means:
+	//       List from _all_ namespaces !!!
+	allPods, podsErr := s.pods.Lister().Pods("").List(labels.Everything())
```

Comment:
it would make sense to use `metav1.NamespaceAll` here to clearly communicate that we're asking for all namespaces

Reply:
Ah, der is a constant for it! Nice.